### DISCUSSION TITLE
Brainstorm sessions 18-6-21

### DISCUSSION THREAD

**Discussion by mschilling** [framna-nl-archive/krew-workspace#121] [Ideas]
Kunnen we eens proberen om binnen 10 minuten zoveel mogelijk ideeën vast te leggen?

Denk bijvoorbeeld aan hoofdonderwerpen:
* core design + assets (hoe komen we daaraan? hoe richten we richten dit in)
* login flow
* functioneel ontwerp (welke schermen, in welke volgorde? -> milestones?)
* systeem documentatie
* performance (wat willen we meten en hoe?)

En natuurlijk mag iedereen _coolstuff_ noemen wat leuk zou zijn om te doen :-)

**Comment by aaltw** [comment]
dark mode
typings.d.ts (in plaats van model interfaces?)

**Reply by aaltw** [reply]
Scratch 'typings.d.ts'.

Reden: de .d.ts files zijn vooral bedoeld als je bijvoorbeeld de api van een .js of een externe library wil interfacen. En voor onze projecten kunnen we (en zouden we) voor `export interface xyz {}` moeten gebruiken.
